.github/workflows/python_tests.yml

```diff
@@ -5,6 +5,12 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  create:
+    branches: [master]
+    tags: ['**']
+  schedule:
+    - cron: "0 4 1 * *"
```

Comment:
runs the CI as a cron job every 1st day of a month at 4am in the night

Reply:
Why not every day though?

Reply:
I thought cause of the environment :grinning: , and we only need to cover the case where mne or numpy change something - that's not so frequent

Reply:
are you happy with this PR otherwise?